src/commands/convert/hdf2asff.ts

```diff
@@ -68,22 +84,25 @@ export default class HDF2ASFF extends Command {
               console.log(result.FailedFindings)
             }
           } catch (error) {
-            console.error(`Failed to upload controls: ${error}`)
+            if (typeof error === 'object' && _.get(error, 'code', false) === 'NetworkingError') {
```

Comment:
Is this cert issue the only time that a NetworkingError will be produced?  If not, then might be worthwhile adjusting the error text to say that using --certificate or --insecure *might* resolve your issue.